jobs/ci-kubernetes-bazel-test.sh

```diff
@@ -25,6 +25,6 @@ bazel clean --expunge
 make bazel-test && rc=$? || rc=$?
 
 # Coalesce test results into one file for upload.
-"$(dirname "${0}")/../images/pull_kubernetes_bazel/coalesce.py"
+"$(dirname "${0}")/../images/bazelbuild/coalesce.py"
```

Comment:
There is no reason, and it's a pain that your PR will fail tests because it can't find `coalesce.py`.